package.json

```diff
@@ -62,7 +62,7 @@
   "dependencies": {
     "@actions/artifact": "^0.5.2",
     "@actions/core": "^1.4.0",
-    "@microsoft/powerplatform-cli-wrapper": "^0.1.41",
+    "@microsoft/powerplatform-cli-wrapper": "^0.1.43",
```

Comment:
that package already exists in pp-cli-wrapper's package feed, but that doesn't have your dist changes from your latest PR microsoft/powerplatform-cli-wrapper#85

Reply:
I'd assume this PR will need updating first